docs/integrations/webhooks/sync-data.mdx

```diff
@@ -87,6 +87,24 @@ pnpm add svix
 
 </CodeBlockTabs>
 
+### Add your endpoint to Middleware
+
+Your Route Handler must be made public or ignored by Middleware to allow the request to succeed. The following example will make any webhooks created under `app/api/webhooks/` public. See [authMiddleware](/docs/references/nextjs/auth-middleware) for more information.
```

Comment:
```suggestion
Your API route must be made public or ignored by Middleware to allow the request to succeed. The following example will make any webhooks created under `app/api/webhooks/` or in `pages/api/webhooks.ts` public. See [authMiddleware](/docs/references/nextjs/auth-middleware) for more information.
```

Changed language to be inclusive of Pages router since we have a Pages example on the page

Reply:
```suggestion
Clerk's `authMiddleware()` protects all routes by default. In order for the webhook to reach your endpoint, you must make the Route Handler public. To do so, you can pass the route to the [`publicRoutes` property](/docs/references/nextjs/auth-middleware#make-routes-public-by-using-public-routes) in the `authMiddleware()` options.

The following example makes the `/api/webhooks` route and all its subroutes public:
```

Reply:
@S3Prototype -- I've changed this to be inclusive, by mentioning both. They are API Routes in Pages Router and Route Handlers in App Router. There isn't a single official term that applies to both.